charactersheet/charactersheet/services/common/account/user_service.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+/**
+ * A global service that fetches the user's account information.
+ */
+var UserServiceManager = new SharedServiceManager(_UserService, null);
+
+/**
+ * An internal service implementation that holds onto data regarding the
+ * account login status, tokens and expiration.
+ *
+ * This service contains all of the relevant information for keeping
+ * track of a user's authentication state.
+ */
+function _UserService(config) {
+    var self = this;
+
+    self.url = '/api/users.json';
```

Comment:
You get them from the API schema. The root URL of the API has the endpoints in it.

```
{
  "users": "url/for/users.json",
  "xmpp": "url/for/xmpp.json"
  ...
}
```

We could fetch the root URL and knowing the identifiers for each type, get the url dynamically.